### ISSUE TITLE
NVDA does not announce combobox and doesn't read combobox content if it is wrapped inside div with role as gridcell

### ISSUE THREAD

**Issue by Snakerman7** [nvaccess/nvda#11348]
I've made a simple HTML with this issue. 
https://jsfiddle.net/yevhen_zmiyenko/ncqLzv10/60/
NVDA + Firefox work fine. They announce about the combo box and read the combo box content.
NVDA + Chrome doesn't work. NVDA doesn't announce the combo box and doesn't read the combo box text. NVDA in Chrome ignores the div with combobox role.

### Actual behavior:
NVDA doesn't announce the combo box and doesn't read the combo box text
### Expected behavior:
NVDA announces the combo box and reads the combo box text
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.1
#### Windows version:
Windows 10 2004 build 19041.329
#### Name and version of other software in use when reproducing the issue:
Google Chrome Version 83.0.4103.116 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2019.3.1
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Adriani90** [comment]
I filled Chromium bug:
https://bugs.chromium.org/p/chromium/issues/detail?id=1435363&q=label%3Achromium-website

@aleventhal I've filled several Chromium bugs with this label, since I didn't find another one which is more a11y adequate. Could you change the label to another more appropriate one to gain some more attention for accessibility?
